homeassistant/helpers/config_validation.py

```diff
@@ -441,6 +441,15 @@ def url(value: Any) -> str:
     raise vol.Invalid('invalid url')
 
 
+def ip_address(value: Any) -> str:
+    """Validate an IP address."""
+    value = str(value).strip()
```

Comment:
Use `string(value)` here. Additionally, I don't think we should strip the whitespaces on the left and right here. It doesn't really make sense to me.